sdk/src/main/java/co/omise/android/ui/InstallmentTermChooserFragment.kt

```diff
@@ -63,13 +106,16 @@ internal class InstallmentTermChooserFragment : OmiseListFragment<InstallmentTer
 
     companion object {
         private const val EXTRA_INSTALLMENT = "InstallmentTermChooserFragment.installment"
+        private const val EXTRA_ZERO_INTEREST_INSTALLMENT = "InstallmentTermChooserFragment.zeroInterestInstallments"
 
-        fun newInstance(installment: PaymentMethod) =
-            InstallmentTermChooserFragment().apply {
-                arguments =
-                    Bundle().apply {
-                        putParcelable(EXTRA_INSTALLMENT, installment)
-                    }
-            }
+        fun newInstance(
+            installment: PaymentMethod,
+            zeroInterestInstallments: Boolean,
```

Comment:
Is `zeroInterestInstallments` needed here?

Reply:
I just noticed that i can access it from somewhere else ...